gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/util/file/DocumentFileUtils.java

```diff
@@ -94,6 +94,20 @@ private void printXML(final Element node, final OutputStream outStream, boolean
         }
     }
 
+    @NotNull
+    public Properties loadExistingProperties(File propertiesFile, String fileName) {
```

Comment:
I think this should be in a `PropertyFileUtils` class because this class is intended for document utilities, so we don't mix unreated things

Reply:
That sounds like a good idea, will move it to a different class